### ISSUE TITLE
🧪  TEST: Make sure documentation links work and point to `latest`

### ISSUE THREAD

**Issue by giovannipizzi** [aiidateam/aiida-tutorials#211]
Since we have changed the documentation, all links are broken.

We should ensure to use the tag v1.3.0 (as soon as it's released) for the links of the current 2020 tutorial (at least).

**Comment by mbercx** [comment]
Since we want to move to having tutorial material that is always up to date with the latest `aiida-core` version, we should check that they point to `latest`, I think.

**Comment by giovannipizzi** [comment]
On the other end, in practice we'll not manage to keep thing updated, and we've had multiple occasions where links were broken?

**Comment by mbercx** [comment]
To reiterate something I've already said in several meetings (but maybe not with you, @giovannipizzi 🙃 ): I think it's _mission-critical_ that the tutorials on the `latest` page are _always_ up to date with the latest AiiDA release. Not just checked every year we do a tutorial. It's the most important part of the documentation, in my view. As highlighted in the ["Documentation system"](https://documentation.divio.com/tutorials/) that we base ourselves on:

_"Tutorials are what will turn your learners into users. A bad or missing tutorial will prevent your project from acquiring new users."_

Although QM and AiiDAlab are useful tools for organising tutorial events, we should also strive to have AiiDA be as easy to install as possible, so people can run the latest version with tutorials that do not break. Again, from the tutorial document:

_"[...]  the single most important thing is that what you ask the beginner to do must work. The learner needs to see that the actions you ask them to take have the effect you say they will have."_

**Comment by giovannipizzi** [comment]
I agree - my only comment is that, from experience, it's something very hard to do :-)

**Comment by mbercx** [comment]
![challengeaccepted](https://user-images.githubusercontent.com/25607960/121677221-e2545100-cab5-11eb-8087-4342f8e8430e.gif)

**Comment by giovannipizzi** [comment]
![Himym GIFs - Find   Share on GIPHY](https://user-images.githubusercontent.com/16132028/121679804-0c5b4280-cab9-11eb-9d4c-39f793f20680.gif)

**Comment by mbercx** [comment]
Why do I feel I'm going to regret this? 😭

**Comment by mbercx** [comment]
@giovannipizzi I've added `linkcheck` to the Sphinx build call in the CI workflow:

https://github.com/aiidateam/aiida-tutorials/blob/2ba4eeae5bfd814d99e280f5526113bf28d825ce/.github/workflows/ci.yml#L42-L43

So this will catch any broken links written in Markdown syntax. `intersphinx` links (preferred) will be caught by the build as well. However, links written in HTML syntax apparently aren't checked. You can find more info here:

https://github.com/aiidateam/aiida-tutorials/wiki/Moving-to-using-MyST-Markdown#links

Feel free to correct in case you see anything wrong. PS: Have you ever tried something like this:

https://www.drlinkcheck.com/

Or one of the [alternatives found here](https://geekflare.com/resource_type/dr-link-check/)?
